### PR TITLE
feat: marketplace auction functionality

### PR DESCRIPTION
**Pull request by cmayorga**
### What changes are being made?

Fixed PR#71 reviews
Changed Market home slider to button based menu
Added logic and UI to start an auction and to see open auctions
Several cosmetics and bugs fixed
Added Random "add card" to work and debug in not mobile devices

### Why are these changes necessary?
To add the auctions feature

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [X] Ready to be merged

### CODE REVIEW

**Review by amacar — CHANGES_REQUESTED** [review]
Please make cleanup of the whole project (remove console logs, non-relevant code comments...) then it is ready to merge :)

File: src/components/HomeMarket.tsx [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    {submenu==="myauctionablecards" && (
+      <AuctionableCards />
+    )}  
+    {submenu==="myauctions" && (
+      <AuctionsMyAuctions />
+    )}      
+    {submenu==="participateinauction" && (
+      <AuctionParticipateIn/>
+    )}  
+    {submenu==="mybids" && (
```
[the remark this answers is not in the document]
Comment (cmayorga) [reply src/components/HomeMarket.tsx:108]: I think this is not definitive, is part of the navigation issues i am working on.

[the remark this answers is not in the document]
Comment (amacar) [reply src/components/HomeMarket.tsx:108]: Noted

File: package.json [hunk 2]
```diff
@@ -22,21 +23,30 @@
     "@ionic-native/screen-orientation": "^5.31.1",
     "@ionic/react": "^5.0.7",
     "@ionic/react-router": "^5.0.7",
-    "@protokol/client": "^1.0.0-beta.39",
-    "@protokol/nameservice-crypto": "^1.0.0-beta",
+    "@opencensus/core": "0.0.22",
```
[the remark this answers is not in the document]
Comment (cmayorga) [reply package.json:26]: @protokol/nameservice-crypto is used at src/store/services/crypto.ts

File: src/components/HomeMarket.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (cmayorga) [reply src/components/HomeMarket.tsx]: I finally did the change to Enum

File: package.json [hunk 3]
```diff
@@ -22,21 +23,30 @@
...
     "date-fns": "^2.19.0",
     "es6-promise-plugin": "^4.2.2",
     "google-map-react": "^2.1.9",
+    "history": "^4.10.1",
+    "ionic": "^5.4.16",
     "ionicons": "^5.0.0",
     "jetifier": "^1.6.6",
+    "moment": "^2.29.1",
+    "node-sass": "^5.0.0",
+    "perf_hooks": "^0.0.1",
```
[the remark this answers is not in the document]
Comment (cmayorga) [reply package.json]: "node-sass": "^5.0.0", is required to allow scss files

**Review by amacar — APPROVED** [review]

**Pull request merged**